OpenMEEG/cmake/UseACML.cmake

```diff
@@ -0,0 +1,5 @@
+if (USE_ACML)
+    set(ACML_ROOT "/home/papadop/Downloads/acml/gfortran64")
```

Comment:
I vote for getting rid of any ACML reference in our code as nobody ever used it in 10 years. If we go with flens we'll have it for free. Until then let's avoid maintaining lines of code for no benefit.